internal/pkg/cli/job_logs.go

```diff
@@ -23,6 +23,9 @@ import (
 
 const (
 	jobAppNamePrompt = "Which application does your job belong to?"
+
+	jobLogNamePrompt     = "Which job's logs would you like to show?"
+	jobLogNameHelpPrompt = "The logs of a deployed job will be shown."
```

Comment:
nit: maybe something like this, bc this is the name prompt?
```suggestion
	jobLogNameHelpPrompt = "The logs of the indicated deployed job will be shown."
```